### DISCUSSION TITLE
Pokemon let's go pikachu

### DISCUSSION THREAD

**Discussion by 13gkd** [skyline-emu/skyline#1396] [General]
![Screenshot_2022-05-28-17-18-16-324_skyline emu](https://user-images.githubusercontent.com/103353628/170824236-78c7680c-65b1-488f-80f9-437529a11b16.jpg)
ghraphical gilitch on sp 845.

**Comment by PixelyIon** [comment]
Don't post these here, we have a [game list repository](https://github.com/skyline-emu/skyline-games-list) for reporting game compatibility issues.

**Reply by 13gkd** [reply]
sorry. thank you for this information
